Sources/PointFree/Account/Change.swift

```diff
@@ -160,15 +168,15 @@ private func fetchSeatsTaken<A>(
     }
 }
 
-private func defaultPricing(for lane: Pricing.Lane, billing: Pricing.Billing) -> Int {
-  switch (lane, billing) {
+private func defaultPricing(for pricing: Pricing) -> Cents<Int> {
```

Comment:
upgraded to cents to strength the code around the prorating logic: https://github.com/pointfreeco/pointfreeco/pull/472/files#diff-bab38e3e97cb96ef710c6056a81b5248R51

Reply:
nice. should we add conversion methods between `Dollar`s and maybe even helpers for formatting to TaggedMoney? We got a lot of mileage out of TaggedTime having them along with `timeInterval`